ndsl/stencils/testing/savepoint.py

```diff
@@ -16,7 +16,7 @@ def dataset_to_dict(ds: xr.Dataset) -> Dict[str, Union[np.ndarray, float, int]]:
 
 def _process_if_scalar(value: np.ndarray) -> Union[np.ndarray, float, int]:
     if len(value.shape) == 0:
-        return value.item()
+        return value.max()  # trick to make sure we get the right type back
```

Comment:
Wow I hate this

Reply:
I plucked this out of a 300+ comment on the numpy git of people trying to kill each other over the fact that `item` gives back a `python` type instead of a `numpy` type.

I stepped away from the fight with my `max` trick, never to open that door again.

Reply:
The hero we need but not the one we deserve